client/application.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import Search from './search'
+import Tweets from './tweets'
+import Store from './store'
+import { Provider } from 'react-redux'
+
+export default class App extends React.Component {
+  render() {
+    return (
+      <div>
+        < Search socket={this.props.socket} />
+        < Provider store={ Store } >
+          < Tweets />
+        < /Provider >
```

Comment:
just a requested style change, remove the space from the bracket and component name:

```
< Search ... />
```

vs.

```
<Search ..../>
```